docs/CONTRIBUTING.md

```diff
@@ -19,11 +19,65 @@ The following fields are required for all plugins included in the `plugins.json`
 - `name` - a human-readable [sentence cased](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) version of the plugin title, for display purposes
 - `package` - the name of the published npm package
 - `repo` - the complete URL to the source repository for the plugin, on GitHub, GitLab, or Bitbucket. All source code must be public, and the repository must allow public issue submissions.
-- `version` - the exact version Netlify will use for all UI-installed instances of the plugin. To update the plugin version later, submit a new pull request to update this field.
+- `version` - the [latest version](#versioning) of the plugin.
 
 ## Update a plugin
 
-You can submit a PR to update the entry for a plugin that you maintain. If you update the plugin `version`, please be sure to test the new version [locally](https://docs.netlify.com/cli/get-started/#run-builds-locally) and [on Netlify](https://docs.netlify.com/configure-builds/build-plugins/#install-a-plugin) before submitting.
+You can submit a PR to update the entry for a plugin that you maintain.
+
+### Versioning
+
+The latest version of the plugin must be specified in the `version` field. When updating it, please be sure to test the new version [locally](https://docs.netlify.com/cli/get-started/#run-builds-locally) and [on Netlify](https://docs.netlify.com/configure-builds/build-plugins/#install-a-plugin) before submitting.
+
+The `version` and `compatibility` fields are used to select the version of plugins installed from the [Netlify plugins directory](https://docs.netlify.com/configure-builds/build-plugins/#ui-installation). This does not apply to plugins installed in the [site's `package.json`](https://docs.netlify.com/configure-builds/build-plugins/#file-based-installation).
```

Comment:
Minor suggestions, though I'm making a bit of an assumption about the package.json installation process. Feel free to edit if necessary.

```suggestion
When a user installs a plugin from the  [Netlify plugins directory](https://docs.netlify.com/configure-builds/build-plugins/#ui-installation), the `version` and `compatibility` fields are used to determine the plugin version that's installed. When a user installs a plugin in a [site's `package.json`](https://docs.netlify.com/configure-builds/build-plugins/#file-based-installation), they can specify a plugin version manually instead.
```